src/config/html.js

```diff
@@ -9,6 +9,12 @@ module.exports = {
     ({ theme_color, background }) => `<meta name="theme-color" content="${theme_color || background}">`,
     ({ appName }) => appName ? `<meta name="application-name" content="${appName}">` : `<meta name="application-name">`
   ],
+  apple: [
```

Comment:
What about do not add new options, and use existing `appleIcon`? For example:
- `appleIcon: true`- generate all icons
- `appleIcon: []` - allow to specify sizes
- `appleIcon: 'minimum'` generate only minimum required icons and tags

Reply:
It looks like moving forward, what you have named `minimum` is actually be the norm and recommended on Apple platforms. So I'd actually go the other way around:

- `appleIcon: 'all'` - generate all icons
- `appleIcon: []` - allow to specify sizes
- `appleIcon: true` - generate only minimum required icons and tags

Reply:
I am afraid changing  logic for `appleIcon: true` is will be breaking change, we can do it in the next major release, not right now

Reply:
Actually, isn't this PR https://github.com/itgalaxy/favicons/pull/312 solving the problem by allowing to pass `appleIcon: ["apple-touch-icon.png"]`? I don't think the generated html is updated though (eg. all sizes are still in html, just images are generated) as I don't see any filter here: https://github.com/itgalaxy/favicons/blob/master/src/config/html.js#L12

Reply:
Feel free to send a fix for #312, `appleIcon: true- generate all icons` is not generate all icons, we can't change it by default